forge-gui/res/cardsfolder/upcoming/ezio_auditore_da_firenze.txt

```diff
@@ -0,0 +1,12 @@
+Name:Ezio Auditore da Firenze
+ManaCost:1 B
+Types:Legendary Creature Human Assassin
+PT:3/2
+K:Menace
+S:Mode$ Continuous | Affected$ Card.Assassin+YouOwn | AffectedZone$ Hand | AddKeyword$ Freerunning:B B | Description$ Assassin spells you cast have freerunning {B}{B}. (You may cast a spell for its freerunning cost if you dealt combat damage to a player this turn with an Assassin or commander.)
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigImmediateTrig | TriggerDescription$ Whenever NICKNAME deals combat damage to a player, you may pay {W}{U}{B}{R}{G} if that player has 10 or less life. When you do, that player loses the game.
+SVar:TrigImmediateTrig:AB$ ImmediateTrigger | ConditionCheckSVar$ LifeAmount | ConditionSVarCompare$ LE10 | Cost$ W U B R G | Execute$ TrigLoseGame | TriggerDescription$ When you do, that player loses the game.
```

Comment:
will the payment not happen with more than 10?

Reply:
UnlessCost maybe

Reply:
Got it working this way, though the trigger does go on the stack even if the attacked opponents life total is 10 or more (but it doesn't allow me to pay)

Reply:
that's how it should be...